### DISCUSSION TITLE
Builds for Fedora, RHEL

### DISCUSSION THREAD

**Discussion by wolf-yuan-6115** [ivangabriele/clamav-desktop#86] [Feature Request Proposals]
As title said.

**Comment by ivangabriele** [comment]
Hi,

I first need to prioritize the next v0.4.0`` release. I'll check afterwards how I plan to provide a Fedora package.

**Reply by beedell-roke** [reply]
> https://github.com/ivangabriele/clamav-desktop/discussions/86#discussioncomment-9709651

@ivangabriele, a COPR repository (or RPMFusion) should be fairly easy, at least in comparison to getting into the main repositories.

**Reply by ivangabriele** [reply]
The issue is not with the distribution but the ability to build and run ClamAV Desktop under Fedora (including dev/testing = the time it takes to setup them and to test it).

I think Tauri v1 was not able to build Fedora-compatible images. I'v recently migrated to the brand new v2 which seems to be able to do that. However, it takes time so I won't do it for the next release.

I plan on supporting Fedora at some point though. With some help it'll be easier once the next version is out.

**Comment by Gideon-Felt** [comment]
As described in the readme:
> (waiting for [tauri-apps/tauri#4402](https://github.com/tauri-apps/tauri/issues/4402
) & [tauri-apps/tauri#5202](https://github.com/tauri-apps/tauri/pull/5202))

(The issue has now been closed, and the pull request merged).

**Reply by beedell-roke** [reply]
@Gideon-Felt, submitted [`pull/215`](https://github.com/ivangabriele/clamav-desktop/pull/215#issue-3699509313).
